client/src/components/Reservation/ReservationTop.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Calendar from './Calendar.jsx';
+import People from './People.jsx';
+import Time from './Time.jsx';
+
+class ReservationTop extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      date: new Date(),
+    };
+  }
+
+  onChange(date) {
+    this.setState({ date });
+  }
+
+  render() {
+    return (
+      <div className="island reservations-sticky-bar--top reservations-sticky-bar hidden">
+        <div className="make-reservation-form-container">
+          <form className="reservation-availability-search-form" name="reservation-availability-search-form">
+            <ul className="inline-layout reservation-fields clearfix reservations-top">
+              <li className="date-picker js-data-picker yform" onClick={this.onCalendar}>
+                <div className="yselect">
+                  <span className="svg-icon">
+                    <i className="far fa-calendar-minus" />
+                  </span>
+                  <input className="date-input js-date-input reservation-input show-calendar" value="Monday, May 27, 2019" />
+                  <span className="yselect_arrow">
+                    <i className="fas fa-caret-down" />
```

Comment:
Lots of unnecessary class names, and mixed naming convention. Keep to a standard naming convention for css selectors. Also, some of these, like 'yselect_arrow' are semantically confusing. I have no idea what it's supposed to select.